### PR TITLE
[FIX] website_sale: Keep salesperson when editing portal partner

### PR DESCRIPTION
**Pull request by simongoffin**
When editing the billing address of an existing portal partner P,
the salesperson of P was changed by the default website salesperson

opw:2649732

### CODE REVIEW

File: addons/website_sale/controllers/main.py [hunk 1]
```diff
@@ -763,13 +763,13 @@ def values_postprocess(self, order, mode, values, errors, error_msg):
                     _logger.debug("website_sale postprocess: %s value has been dropped (empty or not writable)" % k)
 
         new_values['team_id'] = request.website.salesteam_id and request.website.salesteam_id.id
```
Comment (rdeodoo) [inline addons/website_sale/controllers/main.py:765]: Any reason not to do the same with team_id?

**Review by rdeodoo — COMMENTED** [review]
cc @tivisse

**Comment by tivisse** [comment]
@robodoo r+

Thanks.

**Comment by rdeodoo** [comment]
Curious: why 15.0? opw is in 13.0 and code didn't change since then if I am not mistaken?

**Pull request closed without merging**